### ISSUE TITLE
Non-voting membership UI

### ISSUE THREAD

**Issue by johannsell** [Viva-con-Agua/arise#42]
When the implementation of [Drops #256](https://github.com/Viva-con-Agua/drops/issues/256) is done, add a link to the profile page, that allows the user to request the non-voting membership. Show an error, if the preconditions are not fulfilled (checked by Drops). 

Additionally, show a text for explanations (@TobiKaestle) and a status indicator:
(1) No membership
(2) Membership requested
(3) Membership granted
Furthermore, when the membership was granted, an additional text should explain rights and duties to the members. @TobiKaestle we also need this text.

The responsible person of a crew (e.g. Network-ASP) has to get access to a page showing a list of membership applications. The user has to confirm or reject an application from within the list.

**Comment by TobiKaestle** [comment]
New status for the NVM:
- No non voting Membershop
- Has voting Membershop (can only be selected if the supporter has added an address AND it is active in the current crew)

The responsible person of a crew (e.g. Network-ASP) has to get access to a page showing a list of membership applications. The user has to confirm or reject an application from within the list.
Therefore we need a new list with parameters for selecting members of the current crew who have requested the active state in the crew. Also a new component has to be added to alter the links in the list to an AJAX request setting the supporter to active.

**Comment by TobiKaestle** [comment]
@johannsell Yes, you're right, I meant active state with the bottom of my comment, not the NVM. 
And sadly, yes, we need that date, otherwise the informations about the NVM states of supporter will be outdated sooner or later due to supporters not caring about telling us if they don't wan't to stay NVM if the aren't active anymore. IIRC it's even described like this in the articles of association for VcA, but actually I'm not sure about this.
